Documentation/Books/AQL/Views/ArangoSearch/README.md

```diff
@@ -176,94 +176,111 @@ are satisfied.
 
 to match documents which have a 'name' attribute
 
-    FOR doc IN someView SEARCH EXISTS(doc.name)
+    FOR doc IN someView
+      SEARCH EXISTS(doc.name)
```

Comment:
Off-topic: Will `EXISTS(doc.attr)` work with ArangoSearch only, or also in general as a replacement of `HAS(doc, attr)`?

Reply:
EXISTS(...) is an ArangoSearch function which matches documents where the specified attribute (optionally of the specified type) was processed
what are the capabilities of HAS(...)?